### DISCUSSION TITLE
pep621/pdm does not handle PEP735 dependency-groups correctly

### DISCUSSION THREAD

**Discussion by sigma67** [renovatebot/renovate#32940] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

latest

### Please tell us more about your question or problem

minimal reproduction here:

https://github.com/sigma67/renovate-pep621-buildsystem-requires-reproduction/pull/16  
https://github.com/sigma67/renovate-pep621-buildsystem-requires-reproduction/blob/main/renovate.json
https://github.com/sigma67/renovate-pep621-buildsystem-requires-reproduction/blob/main/pyproject.toml

(see failing lockfile artifact update)

https://github.com/renovatebot/renovate/pull/32148 did some great legwork regarding this, but it seems that `pdm.ts` was forgotten (there is an explicit mention of dev-dependencies in there, which is the likely root cause)

fyi @secustor

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by Churro** [comment, marked as answer]
Addressed in PR https://github.com/renovatebot/renovate/pull/32952

**Reply by sigma67** [reply]
thanks!

**Reply by sigma67** [reply]
can confirm that it works correctly now
